lib/game_collection.rb

```diff
@@ -29,6 +29,11 @@ def percentage_visitor_wins
     (visitor_wins.to_f / @games.length).round(2)
   end
 
-
+  def percentage_visitor_wins
+    visitor_wins = @games.count do |game|
+      game.away_goals > game.home_goals
+    end
+    (visitor_wins.to_f / @games.length).round(2)
+  end
 
```

Comment:
Different than how I envisioned (using game_teams) but it passes

Reply:
THANKS ZEKE